packages/components/src/common/toggler/Toggler.tsx

```diff
@@ -0,0 +1,75 @@
+import { useIntl } from 'react-intl';
+
+import { Size, Position } from '..';
+import Button from '../../button/Button';
+import Chevron from '../../chevron/Chevron';
+import CloseButton from '../closeButton/CloseButton';
+
+import messages from './Toggler.messages';
+
+type TogglerProps = {
+  className: string;
+  size: 'sm' | 'md' | 'lg';
+  disabled: boolean;
+  block: boolean;
+  children: React.ReactNode;
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  onClear: () => void;
+  handleKeyDown: (event: React.MouseEvent<HTMLButtonElement>) => void;
+};
+
+export default function Toggler({
+  className,
+  size,
+  disabled,
+  onClick,
+  onClear,
+  handleKeyDown,
+  children,
+  block,
+  ...props
+}: TogglerProps) {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Button
+      className={`btn-${size} btn-input np-toggler np-toggler--${size} ${className}`}
```

Comment:
we should not need `btn-${size} btn-input` and `np-toggler--${size}`

Reply:
please use `classNames` when passing CSS classes to `className` prop

```suggestion
      className={classNames('np-toggler', className})}
```